soccer/src/soccer/ui/main_window.hpp

```diff
@@ -73,6 +73,8 @@ class MainWindow : public QMainWindow {
 
     void setUseRefChecked(bool use_ref);
 
+    rclcpp::Publisher<std_msgs::msg::String>::SharedPtr test_play_pub_;
```

Comment:
nit: pub member shouldn't end in _

Reply:
all other pubs end in _ also (see planner_node.hpp for example) since they are private vars. I can't change that otherwise clang will complain.